packages/@aws-cdk/aws-ivs-alpha/README.md

```diff
@@ -55,6 +55,18 @@ const myChannel = new ivs.Channel(this, 'myChannel', {
 });
 ```
 
+If you want to use RTMP ingest, set `ingestInsecure` property to `true`.
+
+By default, `ingestInsecure` is `false` which means using RTMPS ingest.
```

Comment:
I think the property name is `insecureIngest`

Reply:
Thanks. I’ve fixed.

> Also by default the value is false, so should means not using the RTMP ingest right?

Basically yes, AWS also recommends using RTMP only for specific use cases. The relationship between RTMPS and RTMP is similar to that of HTTPS and HTTP.

RTMP Ingest was added on 2023/3/20. (Ref: https://docs.aws.amazon.com/ivs/latest/LowLatencyUserGuide/doc-history.html)
![IMG_3374](https://github.com/user-attachments/assets/b61a5b74-4d54-43ff-aa36-f8d53a59d86d)

Below is the explanation from the management console. I've added a NOTE to the README based on this information.

![IMG_3373](https://github.com/user-attachments/assets/6bfa15d3-0c6b-4095-81f9-161d3f15f72e)

If this property should not be added, please let me know.
There are cases where the software only supports RTMP, and I want this property. 
However, since it might weaken security, I also think it might be an option to not add this property to L2 and instead use escape hatches if someone wants to use RTMP.